test/bigtest/tests/settings/closing-reasons-test.js

```diff
@@ -1,154 +1,29 @@
-      });
-
-      it('should hide Closing Reason form', () => {
-        expect(closingReasons.addClosingReason.closingReasonForm.isPresent).to.be.false;
-        expect(closingReasons.addClosingReason.isPresent).to.be.true;
-      });
-    });
-
-    describe('cancel action', () => {
-      beforeEach(async () => {
-        await closingReasons.addClosingReason.closingReasonForm.cancelAction();
-      });
-
-      it('should hide Closing Reason form', () => {
-        expect(closingReasons.addClosingReason.closingReasonForm.isPresent).to.be.false;
-        expect(closingReasons.addClosingReason.isPresent).to.be.true;
-      });
-    });
-  });
-
-  it('should display all system reasons', () => {
-    expect(closingReasons.systemReasons().length === defaultReasonsCount).to.be.true;
+    expect(closingReasons.addReasonBtn.isPresent).to.be.true;
   });
 
-  it('should hide actions', () => {
-    closingReasons.systemReasons().forEach(reason => {
-      expect(reason.editAction().length).to.equal(0);
-    });
-  });
-
-  it('should display loaded reasons', () => {
-    expect(closingReasons.reasons().length > defaultReasonsCount).to.be.true;
-  });
-
-  describe('Remove action', () => {
-    beforeEach(async () => {
-      await (
-        new Promise((resolve) => {
-          setTimeout(() => {
-            resolve();
-          }, 1000);
-        })
-      );
-
-      const userReason = find(closingReasons.reasons(), reason => reason.removeAction().length);
-
-      await userReason.removeAction(0).click();
-    });
-
-    it('should refresh items after success', () => {
-      expect(closingReasons.reasons().length > defaultReasonsCount).to.be.true;
-    });
-  });
-
-  describe('Edit action', () => {
-    beforeEach(async () => {
-      await (
-        new Promise((resolve) => {
-          setTimeout(() => {
-            resolve();
-          }, 1000);
-        })
-      );
-
-      const userReason = find(closingReasons.reasons(), reason => reason.editAction().length);
-
-      await userReason.editAction(0).click();
-    });
-
-    it('should open Closing Reason form', () => {
-      expect(closingReasons.closingReasonItem.closingReasonForm.isPresent).to.be.true;
-    });
-
-    describe('submit action', () => {
-      beforeEach(async () => {
-        await closingReasons.closingReasonItem.closingReasonForm.fillValue('test value');
-        await closingReasons.closingReasonItem.closingReasonForm.submitAction();
-      });
-
-      it('should hide Closing Reason form', () => {
-        expect(closingReasons.closingReasonItem.closingReasonForm.isPresent).to.be.false;
-      });
-    });
-
-    describe('cancel action', () => {
-      beforeEach(async () => {
-        await closingReasons.closingReasonItem.closingReasonForm.cancelAction();
-      });
-
-      it('should hide Closing Reason form', () => {
-        expect(closingReasons.closingReasonItem.closingReasonForm.isPresent).to.be.false;
-      });
-    });
```

Comment:
it looks like you've deleted almost all test. Are there any plans/possibilities to add some?

Reply:
Added test for closingReasons, suffixes and prefixes